rust/schema/annotations.py

```diff
@@ -228,41 +228,36 @@ class _:
     loop_body: drop
 
 
-class CallExprBase(Expr):
-    """
-    A function or method call expression. See `CallExpr` and `MethodCallExpr` for further details.
-    """
-    arg_list: optional["ArgList"] | child
-    attrs: list["Attr"] | child
-    args: list["Expr"] | synth
-
-
-@annotate(CallExpr, replace_bases={Expr: CallExprBase}, cfg=True)
+@annotate(CallExpr, cfg=True)
 class _:
     """
-    A function call expression. For example:
+    NOTE: Consider using `Call` instead, as that includes all kinds of calls to
+    functions, and excludes instantiations of tuple structs and tuple enum variants.
+
+    A call expression. For example:
     ```rust
     foo(42);
     foo::<u32, u64>(42);
     foo[0](42);
     foo(1) = 4;
+    Option::Some(42); // tuple enum variant instantiation
```

Comment:
```suggestion
    Option::Some(42); // tuple-like variant instantiation
```
I don't think "enum" belongs between "tuple" and "variant" and I added "like" as per https://doc.rust-lang.org/reference/items/enumerations.html#r-items.enum.tuple-expr.

Reply:
Ok, so the reference is not really consistent in terminology. Either way is fine with me, but I have a slight preference for "tuple(-like) variant" as we use `TupleVariant...` in the class name and I think that phrasing makes it more clear that it's a property of the variant and not the enum.